### DISCUSSION TITLE
安卓连接代理无效，但pc可以

### DISCUSSION THREAD

**Discussion by aliencaocao** [1715173329/UnblockNeteaseMusic#1072] [Q&A] [answered]
### Bug 描述

我自己部署的docker，公网ip，新加坡的服务器。windows可以解锁，但是安卓没用
连的http端口不是https，因为https在pc和手机都无法连接

### 日志内容

无报错，可以看到手机端的流量被路由

### 网易云音乐歌曲链接

https://music.163.com/#/song?id=2019330082

### 网易云音乐版本号

v8.9.70.212604

### 操作系统

miui 14

### 其他信息

不懂安卓需不需要安装ca证书

### 问题排查

- [X] 我确认我使用的核心是由 UnblockNeteaseMusic 项目官方发行，不是其他任何 fork。
- [X] 我确认我已经升级到了最新的核心版本（推荐使用最新构建而不是 release）。
- [X] 我确认我已经启用了 HTTPS 端口。
- [X] 我确认我已经正确设置了 EndPoint。
- [X] 我确认我已经在对应的客户端正确安装了 CA 证书。

**Comment by 1715173329** [comment]
必须安装为系统证书，需要 root 权限。

**Reply by aliencaocao** [reply]
所以安卓现在必须root才能解锁？

**Reply by 1715173329** [reply, marked as answer]
对咯

**Reply by aliencaocao** [reply]
好的谢谢，希望在readme里加入这个
